packages/interactor/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "extends": "@frontside/tsconfig",
   "compilerOptions": {
-    "baseUrl": "/",
     "esModuleInterop": true,
-    "types": ["cypress"]
+    "types": ["cypress"],
+    "sourceMap": false
```

Comment:
🤔

Reply:
I think I'm required to set sourceMap to false in order to run cypress (as seen in this cypress issue [#8477](https://github.com/cypress-io/cypress/issues/8477) so I tried to nest it inside `packages/interactor/test/integrations/cypress/tsconfig.js` but the tsconfig in that directory was not registering.